canonical-web-and-design/maas-ui
# Standardise import and export of actions/reducers/selectors.

## Done
- Standardised import and export of actions/reducers/selectors, so that each entity is in its own folder with its own `index.js`.
- Renamed user and repo actions/reducers/selectors to match the model name from the api (i.e. `users` => `user`, `repositories` => `packagerepository`)
- Moved most actions/reducers/selectors to `app/base` - basically everything except `config` now lives in `base`.

## QA
- Make sure everything works (all the pages load the data, you can save the forms, you can add/edit/delete things etc)
